Description: Using shared library
 Open JTalk uses libhtsengine as static library. In the debian system,
 libhtsengine package provides a shared library. This patch changes
 to use it.
Author: Koichi Akabe <vbkaisetsu@gmail.com>
Last-Update: 2013-12-26
Forwarded: http://sourceforge.net/mailarchive/forum.php?thread_name=20130111191400.2cabb929b72dee8f9f50d12d%40gmail.com&forum_name=hts-engine-users

--- open-jtalk-1.07.orig/configure.ac
+++ open-jtalk-1.07/configure.ac
@@ -231,13 +231,13 @@
             AS_HELP_STRING([--with-hts-engine-library-path],[specify the hts_engine_API library path]),
             [hts_engine_library_path=$withval],
             [hts_engine_library_path='/usr/local/lib'])
-if test ! -f "${hts_engine_library_path}/libHTSEngine.a"; then
-   AC_MSG_ERROR(Cannot find libHTSEngine.a)
+if test ! -f "${hts_engine_library_path}/libHTSEngine.so"; then
+   AC_MSG_ERROR(Cannot find libHTSEngine.so)
 fi
-AC_MSG_CHECKING(for libHTSEngine.a)
-AC_MSG_RESULT($hts_engine_library_path/libHTSEngine.a)
+AC_MSG_CHECKING(for libHTSEngine.so)
+AC_MSG_RESULT($hts_engine_library_path/libHTSEngine.so)
 AC_SUBST(HTS_ENGINE_LIBRARY)
-HTS_ENGINE_LIBRARY=${hts_engine_library_path}/libHTSEngine.a
+HTS_ENGINE_LIBRARY=${hts_engine_library_path}/libHTSEngine.so
 
 # Define HTS_ENGINE_LIBRARY_DIR
 AC_SUBST(HTS_ENGINE_LIBRARY_DIR)
